packages/orbit-components/src/InputGroup/index.tsx

```diff
@@ -288,7 +287,7 @@ const InputGroup = React.forwardRef<HTMLDivElement, Props>(
           </FormLabel>
         )}
 
-        <FakeGroup label={label} error={errorReal} active={active} size={size}>
+        <FakeGroup label={label} error={errorReal} active={active} size={size} disabled={disabled}>
```

Comment:
could you prefix here the props with $? so they do not appear as attributes on FakreGroup